src/utils/UrqlError.re

```diff
@@ -0,0 +1,10 @@
+type t;
+
+[@bs.deriving abstract]
+type error = {
+  [@bs.optional]
+  networkError: Js.Exn.t,
+};
+
+[@bs.new] [@bs.module "urql"]
+external combinedError: string => t = "CombinedError";
```

Comment:
This is just total scaffolding to get it in the right place, but this needs specific work!